styx-scheduler-service/src/test/java/com/spotify/styx/StyxSchedulerTest.java

```diff
@@ -281,7 +280,6 @@ public void testCreateNoopFlyteRunner() {
     assertThat(flyteRunner, notNullValue());
     assertThat(flyteRunner.isEnabled(), is(false));
     assertThrows(FlyteRunner.CreateExecutionException.class, () -> flyteRunner.createExecution(null, null, null, null));
-    assertThrows(FlyteRunner.PollingException.class, () -> flyteRunner.poll(FlyteExecutionId.create("flyte-test","testing","test"), null));
```

Comment:
We can still test this by calling the method without asserting anything.

Reply:
We are a bit over testing here because as long as we can create an instance of `NoopFlyteRunner` and `isEnabled` is `false`, we are good. Testing other behaviour of the class belongs to the unit test of its own.

Reply:
Shall I remove the `CreateExecution` as well?

Reply:
I think we only need to test the instance is of type NoopFlyteRunner, which covers non-null check as well. isEnabled is always false.